com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -3077,6 +3120,9 @@ void RenderGBuffer(CullingResults cull, HDCamera hdCamera, ScriptableRenderConte
                 DrawOpaqueRendererList(renderContext, cmd, hdCamera.frameSettings, rendererList);
 
                 m_GbufferManager.BindBufferAsTextures(cmd);
+#if ENABLE_VIRTUALTEXTURES
+                cmd.ClearRandomWriteTargets();
```

Comment:
I don't understand why there is a call here to ClearRandomWriteTargets, I expect it is dead code. But let's discuss in our meeting about it